diff -uNr a/device/samsung/t0lteskt/selinux/file.te b/device/samsung/t0lteskt/selinux/file.te
--- a/device/samsung/t0lteskt/selinux/file.te	2014-04-13 18:27:19.000000000 +0800
+++ b/device/samsung/t0lteskt/selinux/file.te	2014-04-13 19:05:20.403357009 +0800
@@ -4,7 +4,6 @@
 type firmware_camera, file_type;
 
 type qmuxd_socket, file_type;
-type camera_data_file, file_type, data_file_type;
 type kickstart_data_file, file_type, data_file_type;
 type sensors_data_file, file_type, data_file_type;
 type volume_data_file, file_type, data_file_type;

